### DISCUSSION TITLE
Whatsapp web monitor addin does not work anymore since yesterday

### DISCUSSION THREAD

**Discussion by chrbay68** [rizwansoaib/whatsapp-monitor#207] [General]
The Add-in does not launch anymore in whatsapp web. After starting the addin and clicking ok, there is nothing in the whatsapp web top bar. It started since yesterday. Anybody else experienced this?

**Comment by rizwansoaib** [comment]
@chrbay68 Sorry I have no laptop or system since 3 months so I was not able to update this repo currently I am responding from Mobile device(Android) I hope anyone can debug and send me or he should make a pull request for this.

**Reply by chrbay68** [reply]
I am experiencing this on the following OS:
Edition:	Windows 10 Pro
Version:	22H2
Installed on:	‎6/‎27/‎2022
OS build:	19045.3324
Experience: Windows Feature Experience Pack 1000.19041.1000.0

Browser:
Updating Chrome
Version 116.0.5845.110 (Official Build) (64-bit)

**Comment by rizwansoaib** [comment]
> The Add-in does not launch anymore in whatsapp web. After starting the addin and clicking ok, there is nothing in the whatsapp web top bar. It started since yesterday. Anybody else experienced this?

Please create issue in repo
